tgg/seqr_scripts/pull_seqr_pedigrees.py

```diff
@@ -21,27 +25,43 @@ def pull_project_peds(session_id: str, projects: set):
     """
     with requests.Session() as s:
         # Initialize session with authenticated cookie
-        s.cookies.set_cookie(requests.cookies.create_cookie('sessionid', session_id))
-
-        with open("seqr_pedigrees.txt", "w") as final_ped, open("projects_not_pulled.txt", "w") as errors:
+        s.cookies.set_cookie(requests.cookies.create_cookie("sessionid", session_id))
+        with open("seqr_pedigrees.txt", "w") as final_ped, open(
+            "projects_not_pulled.txt", "w"
+        ) as errors:
             final_ped.write(
                 "Project_GUID\tFamily_ID\tIndividual_ID\tPaternal_ID\tMaternal_ID\tSex\n"
             )
             for project_guid in projects:
                 project_guid = project_guid.rstrip()
-                r = s.get(
-                    f"{SEQR_URL}/api/project/{project_guid}/details"
+                family_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/{GET_FAMILIES_URL}"
                 )
-
-                if r.status_code != requests.codes["ok"]:
-                    logger.info(f"Could not find {project_guid} in seqr")
+                individual_r = s.get(
+                    f"{SEQR_URL}/api/project/{project_guid}/{GET_INDIVIDUALS_URL}"
+                )
+                if (family_r.status_code != requests.codes["ok"]) or (
+                    individual_r.status_code != requests.codes["ok"]
+                ):
+                    if family_r.status_code != requests.codes["ok"]:
+                        logger.info(
+                            f"Could not get family IDs for {project_guid} in seqr."
+                        )
+                    if individual_r.status_code != requests.codes["ok"]:
+                        logger.info(
+                            f"Could not get individual IDs for {project_guid} in seqr."
+                        )
                     errors.write(f"{project_guid}\n")
                 else:
-                    fams = r.json()["familiesByGuid"]
+                    # family_r is stuctured with a high level field familiesByGuid, which contains fields for individual families, which themselves contain various fields, like analysisStatus, analyzedBy, etc.
+                    # importantly, familiesByGuid has fields familyGuid, which is a more keyword type family name, and familyId, which is the family name used in seqr
```

Comment:
```suggestion
                    # `family_r` contains a high level field `familiesByGuid`,
                    # which contains fields for individual families.
                    # The fields within `familiesByGuid` also contain various fields (e.g., analysisStatus, analyzedBy, etc).
                    # Importantly, familiesByGuid has the fields `familyGuid` (keyword type family name) 
                    # and `familyId`, which is the family name used in seqr
```
(optional, slight rewording/reformatting)